docs/tutorialkit.dev/src/content/docs/guides/overriding-components.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Overriding Components'
+description: "Override TutorialKit's default components to fit your needs."
+---
+import { Image } from 'astro:assets';
+import uiTopBar from './images/ui-top-bar.png';
+
+TutorialKit's default components are customizable with [theming](/reference/theming/) options.
+In cases where theming is not enough you can override built-in components with your own ones.
+
+## Configuration
+
+Define `components` option in your `astro.config.ts`:
+
+```ts
+import tutorialkit from '@tutorialkit/astro';
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  integrations: [
+    tutorialkit({
+      components: {
+        // Component overrides go here
+      },
+    }),
+  ],
+});
+```
+
+For example to override `TopBar`, provide a path to the new component:
+
+```ts
+tutorialkit({
+  components: {
+    TopBar: './src/components/CustomTopBar.astro',
+  },
+}),
+```
+
+See [User Interface Reference](/guides/ui/) for component reference.
```

Comment:
Is that the correct link? This page describes the user interface but does not provide a reference.

Reply:
That page describes what `Top Bar` is. But I've also added the same picture on this page so I guess we can remove this link completely.

Reply:
Yeah let's remove it